pkg/cloud/vsphere/services/govmomi/net/net.go

```diff
@@ -97,3 +92,25 @@ func GetNetworkStatus(
 
 	return allNetStatus, nil
 }
+
+// IsExternalIPAddr returns a nil error if the provided IP address is valid and
+// can be used externally to the VM.
+func IsExternalIPAddr(addr string) error {
```

Comment:
`IsExternalIPAddr` can be confused with publically external IP, prefer something like "IsNonLocalIPAddr", thoughts?

Reply:
Hi @andrewsykim,

Maybe, but the docs are pretty clear I think. If I have to push a code change, I'll consider a name change.

Reply:
sure

Reply:
Hi @andrewsykim and @sidharthsurana,

It's been renamed.